test/tests/api/v1_1/auth_tests.py

```diff
@@ -0,0 +1,34 @@
+from config.api1_1_config import config
+from modules.logger import Log
+from modules.auth import Auth
+from on_http_api1_1 import NodesApi as Nodes
+from proboscis.asserts import assert_equal
+from proboscis import before_class
+from proboscis import after_class
+from proboscis import test
+
+LOG = Log(__name__)
+
+@test(groups=['auth.tests'])
+class AuthTests(object):
+
+    def __init__(self):
+        self.__client = config.api_client
+
+    @before_class()
+    def setup(self):
+        """setup test environment"""
+        Auth.enable()
+
+    @after_class(always_run=True)
+    def teardown(self):
+        """ restore test environment """
+        Auth.disable()
+
+    @test(groups=['test-nodes-withauth'])
+    def test_nodes(self):
```

Comment:
Agree. The existing tests should be reused. Maybe run twice of those tests. 1st time no auth. 2nd time with auth. The `modules/auth.py` is simply handle this with `Auth.enable()` and `Auth.disable()`. I'll figure out how to apply this to existing tests later. 
For this PR, just try to run a simple case to make sure auth works.